README.md

```diff
@@ -2,11 +2,25 @@
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S1-CS2103T-W16-3/tp/actions)
 
 ## Overview
-* Private tutors often have many students and have difficulty keeping track of each of the students and their details.
-* TutorAid provides value through the organisation of such data.
-* TutorAid can also help these tutors with the tedious task of tracking every student's study progress and any upcoming milestones.
+**TutorAid is a desktop application for private tutors to manage the details of their students and lessons.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+
+Want to download and start using TutorAid? Head over to the [_Quick Start_ section of the **User Guide**](https://ay2122s1-cs2103t-w16-3.github.io/tp/UserGuide.html#3-quick-start) to start your TutorAid experience!
 
 ![Ui](docs/images/Ui.png)
 
+## About Us
+
+To learn more about our team, head over to the [**About Us**](AboutUs.html) page.
+
+## User Guide
+
+If you are interested in using TutorAid, refer to the [**User Guide**](UserGuide.html).
+
+## Developer Guide
+
+If you are interested about developing TutorAid, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
```

Comment:
Minor nit-pick but can I check if it should be `interested in` or `interested about`? 😅